### ISSUE TITLE
Problems with heat-inserting threaded inserts in docking module holder

### ISSUE THREAD

**Issue by SusanL82** [Coen-Lab/chronic-neuropixels#11]
Hi everyone,

We've since found a workaround, but I'm posting here anyway in case it helps someone: 

We've been having problems with heat-inserting threaded inserts in the docking holder for the NP1 probe. Basically the walls of the insert-hole kept cracking when we tried to insert the metal parts. 

This was particularly a problem because we can currently not print resin parts in the lab, and our printing service is quite expensive. 

Initially we used threaded inserts for M3 screws that had a 5mm outer diameter. These completely destroyed the plastic when inserted.
We then switched to the M2.5 threaded inserts from CNCkitchen, which have a 4.6mm outer diameter and are 4mm long. These inserts fit much better, but still cracked the plastic (see pictures). And you can see that there is very little melting around the metal parts. 

![insert1](https://github.com/Coen-Lab/chronic-neuropixels/assets/78316402/53523bb9-6f1c-4fec-a535-522bfe3eca71)
![insert2](https://github.com/Coen-Lab/chronic-neuropixels/assets/78316402/e1644f34-44dd-4b21-887f-352fbf1938d9)

So, wondering if everything I thought I knew about plastic was wrong, I then tried to melt one of our broken docking holders with a hot soldering iron. I could not. The hole-walls remained solid for a long time and at the point where the plastic became somewhat flexible it just lost structural integrity and cracked/tore in a similar way that happened with our inserts. 
I think this is because resin is not a thermoplast (at least our printed resin does not act as a thermoplast). I was able to make a small scorch mark with a very hot soldering iron (approx 400C), but there was no melting. 

We ended up just gluing/reinforcing the cracks in our best docking holder with some 5-minute epoxy and this seems to hold up fine for now. 

Of note: 
We were able to insert the larger inserts in the payload holder just fine.  Similarly, we're able to insert the tiny inserts in the payload module without too much trouble. I think the larger/wider amount of plastic around the holes keeps things from cracking.

Best, 
Susan

**Comment by SusanL82** [comment]
Actually, I think I can close this myself :)

**Comment by pipcoen** [comment]
Hi @SusanL82,
This is surprising because we've never had this issue before (and neither have other labs as far as I know). Could you just confirm the material you printed these pieces in for future reference? Typically we use M3 threads with a 5.5mm outer diameter... I wonder if the company that printed your files (or perhaps an error in a file that was our fault?!) cause the holes to be printed too small,

Best,

Pip

**Comment by SusanL82** [comment]
I also don't now anyone who has these problems before (I certainly never encountered them with PLA parts). 
The holes in our prints had a diameter of a little under 4mm. We downloaded the STL files a while ago and ended up printing the parts in September last year. 
The docking holder fits fine around the metal bars, so I guess only some holes came out small. 

I checked and our printing company made the docking holder and payload holder out of PA12, which I believe is a type of nylon. 
I remembered wrong earlier and we only made the smaller modules out of resin. I've edited my original post to reflect this.  

I really don't know what happened (the not-melting part is very strange to me, but maybe nylon has a much higher melting point than other plastics?). I hope that somehow this is helpful for reference at least.
